.travis.yml

```diff
@@ -60,14 +12,90 @@ before_install:
   - chmod +x docker-compose
   - sudo mv docker-compose /usr/local/bin
 
-before_script:
-  - if [ "$CC" == "gcc" ]; then export CC=gcc-4.8; fi
-  - if [ "$CXX" == "g++" ]; then export CXX=g++-4.8; fi
-  - if [ "$CC" == "clang" ]; then export CC=clang-3.8; fi
-  - if [ "$CXX" == "clang++" ]; then export CXX=clang++-3.8; fi
-  - docker-compose build base
-  - if [ "$TARGET" == "android" ]; then docker-compose build java; fi
-  - docker-compose build ${TARGET}
-
-script:
-  - docker-compose run ${TARGET}
+jobs:
+  fast_finish: true
+  allow_failures:
+    - env: TARGET=sonar-scan
```

Comment:
Could you please elaborate why this is set to allow failures? We generally don't want to have an unscanned code in,